Documentation/Scripts/fetchRefs.sh

```diff
@@ -6,43 +6,49 @@ ALLBOOKS="HTTP AQL Manual Cookbook Drivers"
 
 GITAUTH="$1"
 
-for book in ${ALLBOOKS}; do 
+for book in ${ALLBOOKS}; do
 
     repos=$(grep '^ *<!-- SYNC: ' "../Books/${book}/SUMMARY.md" |sed -r 's;^ *<!-- SYNC: (.+) -->$;\1;')
+    reposUnique=$(awk '!seen[$0]++' <(for oneRepo in ${repos}; do echo "${oneRepo}"; done))
```

Comment:
What does that do? Why don't you simply use `sort -u`? It looks like you would not need the loop to get things on different lines if proper escaping would be used.

Reply:
Removes duplicates without sorting (so not changing order, and said to be faster), found here: https://stackoverflow.com/a/21201501/2044940

How can the loop be avoided?
BTW. I added the missing quotes to `${repos}` in the second commit: https://github.com/arangodb/arangodb/pull/8429/commits/db87711263aa2d0634daa15068cb0029a7934651